pages/popular.js

```diff
@@ -29,12 +29,14 @@ export default function Popular({ data }) {
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <div className="mx-auto container">
-        <h1 className="text-4xl font-bold mb-4">Popular Profiles</h1>
-        <ul>
+      <div className="p-5">
+        <h1 className="lg:text-3xl md:text-2xl text-xl font-bold mb-4 w-fit p-3 text-[#202328] rounded border-b-4 border-b-green-500">
```

Comment:
The green is not consistent with other pages and the brand - but this can be removed after the merge